vip-plugins/vip-plugins.php

```diff
@@ -0,0 +1,222 @@
+<?php 
+
+/**
+ * Retrieve featured plugins from the vip.wordpress.com API
+ *
+ * @return array an array of plugins
+ */
+function wpcom_vip_fetch_vip_featured_plugins() {
+	$plugins = wp_cache_get( 'wpcom_vip_featured_plugins' );
+
+	if ( false === $plugins ) {
+		$plugins = array();
+		$url_for_featured_plugins = 'https://vip.wordpress.com/wp-json/vip/v1/plugins?type=technology';
+		$response = vip_safe_wp_remote_get( $url_for_featured_plugins, false, 3, 5 );
+
+		if ( ! $response || is_wp_error( $response ) ) {
+			trigger_error( 'The API on vip.wordpress.com is not responding (' . esc_url( $url_for_featured_plugins ) . ')' );
+			return false;
+		}
+
+		$plugins = json_decode( $response['body'] );
+
+		if ( empty( $plugins ) ) {
+			return false;
+		}
+
+		wp_cache_set( 'wpcom_vip_featured_plugins', $plugins, '', HOUR_IN_SECONDS * 4 );
+	}
+
+	return $plugins;
+}
+
+/**
+ * Render the featured partner plugins to the plugins screenReaderText
+ * Uses the notice hooks as that is all we have on these pages
+ *
+ * @return null
+ */
+function wpcom_vip_render_vip_featured_plugins() {
+	$screen = get_current_screen();
+
+	if ( ! ( 'plugins' === $screen->id || 'plugins-network' === $screen->id ) ) {
+		return;
+	}
+
+	$plugins = wpcom_vip_fetch_vip_featured_plugins();
+
+	if ( ! $plugins ) {
+		return;
```

Comment:
Should we show the user an error? Might be confusing if featured plugins suddenly don't show up when we run into an error.